gdp/dsda/dsda_functions.py

```diff
@@ -11,7 +11,7 @@
 from gdp.dsda.model_serializer import StoreSpec, from_json, to_json
 from pyomo.common.errors import InfeasibleConstraintException
 from pyomo.contrib.fbbt.fbbt import fbbt
-from pyomo.contrib.gdpopt.data_class import MasterProblemResult
+# from pyomo.contrib.gdpopt.data_class import MasterProblemResult
```

Comment:
If this is not imported anywhere, do not just comment it out but remove it

Reply:
Okay, I removed the imports